rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -2941,7 +2916,7 @@ bool ppu_interpreter::B(ppu_thread& ppu, ppu_opcode_t op)
 bool ppu_interpreter::MCRF(ppu_thread& ppu, ppu_opcode_t op)
 {
 	CHECK_SIZE(ppu_thread::cr, 32);
-	reinterpret_cast<u32*>(ppu.cr)[op.crfd] = reinterpret_cast<u32*>(ppu.cr)[op.crfs];
+	reinterpret_cast<u32*>(&ppu.cr)[op.crfd] = reinterpret_cast<u32*>(&ppu.cr)[op.crfs];
```

Comment:
Don't reinterpret pointer to the struct, it's more correct to reinterpret `+ppu.cr.bits`.